fineract-provider/src/main/java/org/apache/fineract/portfolio/collectionsheet/api/CollectionSheetApiResourse.java

```diff
@@ -75,8 +72,11 @@ public CollectionSheetApiResourse(final CollectionSheetReadPlatformService colle
     @POST
     @Consumes({ MediaType.APPLICATION_JSON })
     @Produces({ MediaType.APPLICATION_JSON })
-    public String generateCollectionSheet(@QueryParam("command") final String commandParam, final String apiRequestBodyAsJson,
-            @Context final UriInfo uriInfo) {
+    @ApiOperation(value = "Generate Individual Collection Sheet | Save Collection Sheet", httpMethod = "POST", notes = "Generate Individual Collection Sheet:\n" + "This Api retrieves repayment details of all individual loans under a office as on a specified meeting date.\n" + "Save Collection Sheet:\n" + "This Api allows the loan officer to perform bulk repayments of individual loans and deposit of mandatory savings on a given meeting date.")
```

Comment:
add new line character "\n"
and mention about the request/response in the notes section itself